tools/occlum

```diff
@@ -69,6 +69,8 @@ Usage:
         All runtime dependencies required by the Occlum instance---except Intel SGX driver,
         enable_rdfsbase kernel module, and Intel SGX PSW---are included in the package.
         If package_name is not specified, the directory name of Occlum instance will be used.
+        In default only HW mode package is supported. Simulation mode package could be supported
```

Comment:
"By default, only HW release mode package is supported"
The below implementation does not check the HW debug mode when the "debug" flag is not set.